.github/workflows/publish-nightly.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           NEXUS_USER: ${{ secrets.NEXUS_USER }}
           NEXUS_PW: ${{ secrets.NEXUS_PW }}
 
-      - name: Publish Gradle Plugin to Apache Nexus Repository
```

Comment:
This extra info is irrelevant since entire plugin publishes to Apache Nexus repo